src/Component/Modal/CartItemModalContent.tsx

```
@@ -0,0 +1,104 @@
+    }
+  };
+  return (
+    <Modal.Container>
+      <Modal.Overlay />
+      <Modal.Content position="bottom">
+        <Modal.Title>장바구니</Modal.Title>
+        <Modal.Body>
+          <StyledUl>
+            {cartItems.map((item) => (
+              <StyledLi>
+                <DeleteButton onClick={() => handleDeleteClick(item.id)}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)